### DISCUSSION TITLE
heroku deploy django app

### DISCUSSION THREAD

**Discussion by Ybouali** [community/community#116304] [Programming Help]
### Select Topic Area

Question

### Body

i have been done working on a app in django and i deployed this app to heroku but i get this error can some here help me to fix this error 
<img width="1556" alt="Screenshot 2024-04-01 at 01 04 26" src="https://github.com/community/community/assets/89746106/70890431-bdab-46c0-80ab-a58a1be83d01">

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Ybouali ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by Ybouali** [reply]
okay thank you
